functions/go/set-namespace-legacy/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.14
```

Comment:
Why keep the legacy function around?

Reply:
@mengqiy

Reply:
Initially I thought we were doing this in 2 steps:
1) we rename the old one as "legacy" and meanwhile setup the new one.
2) remove the legacy after the new one is ready (including code, test and example and README).

We also remove the old one in this PR if you want to do it.